nm/src/nm.c

```diff
@@ -404,8 +404,11 @@ _j4status_nm_device_update(J4statusPluginContext *context, J4statusNmSection *se
                 if ( context->formats.up_wifi_tokens & TOKEN_FLAG_UP_WIFI_SSID )
                 {
                     GBytes *raw_ssid = nm_access_point_get_ssid(section->ap);
-                    data.ssid = g_variant_new_take_string(g_strndup(g_bytes_get_data(raw_ssid, NULL),
-                                                                    g_bytes_get_size(raw_ssid)));
+                    if (raw_ssid) {
```

Comment:
This condition must be in the previous commit as it’s the one introducing the `NULL` case.

Nit: also 3 style issues here: spaces around parens, `!= NULL` and bracket on the following line